pandas/conftest.py

```diff
@@ -1202,6 +1203,39 @@ def utc_fixture(request):
 utc_fixture2 = utc_fixture
 
 
+@pytest.fixture(
+    params=[
+        pytest.param(None, id=str(locale.getlocale())),
+        "it_IT.utf8",
+        "zh_CN",
```

Comment:
Maybe 

```suggestion
        "zh_CN",  # Note: encoding is automatically 'gb2312'
```

Reply:
I think it would be good to add `zh_CH.utf8` and `zh_CH.gb2312` explicitly

Reply:
Ok, I'll add `zh_CH.utf8`.
Strangely enough, `zh_CN.gb2312` did not seem to be working when I tried on my local windows. I'll have a look again..

Reply:
I will also add `it_IT` by the way, so that both locales have their utf8 AND non-utf8 encoding activated in the test

Reply:
done